### DISCUSSION TITLE
Files sync, but do not display on the reader

### DISCUSSION THREAD

**Discussion by PeterBerthelsen** [fsantini/KoboCloud#148] [Q&A]
Followed the install instructions for Gdrive and everything seems to work fine for grabbing the files. 
However, on my homescreen, none of the books that are pulled in are showing. Am I missing something?

.add\kobocloud\kobocloudrc has the gdrive link
.add\kobocloud\Library shows all .epub books in the folder specified. 

I've restarted the Kobo, and the new books are not showing up in my Books section.

**Comment by PeterBerthelsen** [comment]
I'm using the Kobo Clara 2E model.

**Comment by DenizAsker** [comment]
Hi,
Same issue here using Box on a Kobo Aura.
I am using the latest firmware without any addon apart from Kobo Cloud.

I can see the epub are properly downloaded in .add/kobocloud/library but they do not show up even restarting or manually syncing multiple times.

After reconnecting, if I move them to the standard folder with the windows explorer, the import taskbar appears immediately when I disconnect the reader from the computer and they show up properly.

I am avaiilable for testing or if other information is needed.

Thank you!

**Comment by DenizAsker** [comment]
Hi again,

I continued to search about this issue and I managed to make it work.

Refering to this issue : [https://github.com/fsantini/KoboCloud/issues/153]
I understood that the Kobo OS could be the culprit.

So, I decided to save the books, remove the SD Card and make a firmware reset going in => Settings>Device Information.
I did not use the clip manual reset.

After that, I reconfigured my now blank Aura reader leaving the SD Card away as the internal memory is probably sufficient for my use and hoping I could easily add stuff thanks to Kobo Cloud (purging everything from time to time only).

I then reinstalled Kobo Cloud, putting the tgz file in the .kobo folder.
I restarted, connected to the pc, added my Box shared folder link in .add/kobocloud/kobocloudrc.

I shutdown, restarted again, the sync with Box worked as expected.
Then, I waited for one or two minutes, made a new manual sync and the import taskbar appeared.

So it works !

To the Kobo Cloud team :
You may recommend to do a firmware reset prior to the Kobo Cloud install for new users.

Thank you very much for this mod ! It is the one thing that was missing to the Kobo Aura and you managed to add it !
Congratulations !


Edit : I add a few more books on my Box Shared folder and they showed up immediately after a manual sync ! Great !
The only minor issue is that the covers are not displayed... but at least, the books are now visible and fully readable.
